drivers/interrupt_controller/intc_ioapic.c

```diff
@@ -108,8 +108,8 @@ __pinned_bss
 uint32_t ioapic_suspend_buf[SUSPEND_BITS_REQD / 32] = {0};
 #endif
 
-static uint32_t __IoApicGet(int32_t offset);
-static void __IoApicSet(int32_t offset, uint32_t value);
+static uint32_t __IoApicGet(uint32_t offset);
+static void __IoApicSet(uint32_t offset, uint32_t value);
```

Comment:
all these `int32_t` to `uint32_t` type changes seem unnecessary.

Reply:
Ok, recovery the change here.